internal/ingress/store/store_test.go

```diff
@@ -49,6 +50,10 @@ func Test_networkingIngressV1Beta1(t *testing.T) {
 			name: "correctly transformers from extensions to networking group",
 			args: args{
 				obj: &extensions.Ingress{
+					TypeMeta: metav1.TypeMeta{
+						APIVersion: "extensions/v1beta1",
+						Kind:       "Ingress",
+					},
 					Spec: extensions.IngressSpec{
```

Comment:
Since we are resorting to JSON, can you please add in a test for `Status` as well?

Reply:
Done